gsa/src/web/pages/reports/detailspage.js

```diff
@@ -714,6 +715,16 @@ const load = ({
     filter = reportFilter;
   }
 
+  if (!hasValue(filter)) {
+    // use filter from store
+    filter = pageFilter;
+  }
```

Comment:
I am not 100 percent sure if these lines are really necessary because the page filter will be included in the defaultFilter by the FilterProvider already. Currently this overrides a possible `?filter=...` url parameter (named locationQueryFilter in FilterProvider).